### PR TITLE
net-tools: Add net-tools package.

### PR DESCRIPTION
**Pull request by AkitoTakada**
I confirmed the commands included in net-tools.

```
root@qemuarm64:~# netstat -a
Active Internet connections (servers and established)
Proto Recv-Q Send-Q Local Address Foreign Address State
tcp 0 0 0.0.0.0:http 0.0.0.0:* LISTEN
Active UNIX domain sockets (servers and established)
Proto RefCnt Flags Type State I-Node Path
warning, got bogus unix line.
warning, got bogus unix line.
warning, got bogus unix line.
warning, got bogus unix line.
warning, got bogus unix line.
root@qemuarm64:~# netstat -r
Kernel IP routing table
Destination Gateway Genmask Flags MSS Window irtt Iface
default 10.0.2.2 0.0.0.0 UG 0 0 0 eth0
10.0.2.0 0.0.0.0 255.255.255.0 U 0 0 0 eth0
```

```
root@qemuarm64:~# ifconfig
eth0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST> mtu 1500
inet 10.0.2.15 netmask 255.255.255.0 broadcast 10.0.2.255
ether 52:54:00:12:35:02 txqueuelen 1000 (Ethernet)
RX packets 2 bytes 1180 (1.1 KiB)
RX errors 0 dropped 0 overruns 0 frame 0
TX packets 1 bytes 590 (590.0 B)
TX errors 0 dropped 0 overruns 0 carrier 0 collisions 0
```

```
lo: flags=73<UP,LOOPBACK,RUNNING> mtu 65536
inet 127.0.0.1 netmask 255.0.0.0
loop txqueuelen 1000 (Local Loopback)
RX packets 12 bytes 972 (972.0 B)
RX errors 0 dropped 0 overruns 0 frame 0
TX packets 12 bytes 972 (972.0 B)
TX errors 0 dropped 0 overruns 0 carrier 0 collisions 0
```

```
root@qemuarm64:~# ipmaddr
1: lo
inet 224.0.0.1
2: eth0
link 01:00:5e:00:00:01
inet 224.0.0.1
```

```
root@qemuarm64:~# iptunnel -V
net-tools 2.10-alpha
Alexey Kuznetsov
```

```
root@qemuarm64:~# mii-tool -V
net-tools 2.10-alpha
David Hinds based on Donald Becker's mii-diag
```

```
root@qemuarm64:~# route
Kernel IP routing table
Destination Gateway Genmask Flags Metric Ref Use Iface
default 10.0.2.2 0.0.0.0 UG 0 0 0 eth0
10.0.2.0 0.0.0.0 255.255.255.0 U 0 0 0 eth0
```

### CODE REVIEW

File: recipes-debian/net-tools/net-tools_debian.bb [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+do_patch[depends] += "quilt-native:do_populate_sysroot"
+
+# The Makefile is lame, no parallel build
+PARALLEL_MAKE = ""
+
+do_unpack[cleandirs] += "${S}"
+
+# We invoke base do_patch at end, to incorporate any local patch
+python do_patch() {
```
Comment (masami256) [inline recipes-debian/net-tools/net-tools_debian.bb]: @AkitoTakada Can this function remove ? poky's recipe calls nettools_do_patch in do_patch task, so it needs to call patch_do_patch. However, this recipe doesn't call nettools_do_patch, so it looks as if it's okay to remove this function.

**Comment by masami256** [comment]
@AkitoTakada please fix following warning.

```
WARNING: net-tools-1.60+git20180626.aebd88e-r0 do_patch: Fuzz detected:                                                                                                                               
                                                                                                                                                                                                      
Applying patch musl-fixes.patch                                                                                                                                                                       
patching file lib/inet_sr.c                                                                                                                                                                           
Hunk #1 succeeded at 28 (offset 2 lines).                                                                                                                                                             
patching file lib/util-ank.c                                                                                                                                                                          
patching file netstat.c                                                                                                                                                                               
Hunk #1 succeeded at 89 with fuzz 2 (offset 2 lines).                                                                                                                                                 
patching file slattach.c                                                                                                                                                                              
Hunk #1 succeeded at 45 with fuzz 1 (offset 1 line).                                                                                                                                                  
                                                                                                                                                                                                      
The context lines in the patches can be updated with devtool:                                                                                                                                         
                                                                                                                                                                                                      
    devtool modify net-tools                                                                                                                                                                          
    devtool finish --force-patch-refresh net-tools <layer_path>                                                                                                                                       
                                                                                                                                                                                                      
Don't forget to review changes done by devtool!                                                                                                                                                       
```

**Comment by masami256** [comment]
@AkitoTakada would you fix following warnings please.
```
WARNING: net-tools-1.60+git20180626.aebd88e-r0 do_package: net-tools: alternative target (/usr/share/man/man1/dnsdomainname.1 or /usr/share/man/man1/dnsdomainname.1.net-tools) does not exist, skippi
ng...
WARNING: net-tools-1.60+git20180626.aebd88e-r0 do_package: net-tools: alternative target (/sbin/arp or /sbin/arp.net-tools) does not exist, skipping...
WARNING: net-tools-1.60+git20180626.aebd88e-r0 do_package: net-tools: alternative target (/sbin/ifconfig or /sbin/ifconfig.net-tools) does not exist, skipping...
WARNING: net-tools-1.60+git20180626.aebd88e-r0 do_package: net-tools: alternative target (/sbin/plipconfig or /sbin/plipconfig.net-tools) does not exist, skipping...
WARNING: net-tools-1.60+git20180626.aebd88e-r0 do_package: net-tools: alternative target (/sbin/rarp or /sbin/rarp.net-tools) does not exist, skipping...
WARNING: net-tools-1.60+git20180626.aebd88e-r0 do_package: net-tools: alternative target (/sbin/route or /sbin/route.net-tools) does not exist, skipping...
WARNING: net-tools-1.60+git20180626.aebd88e-r0 do_package: net-tools: alternative target (/sbin/slattach or /sbin/slattach.net-tools) does not exist, skipping...
WARNING: net-tools-1.60+git20180626.aebd88e-r0 do_package: net-tools: alternative target (/bin/dnsdomainname or /bin/dnsdomainname.net-tools) does not exist, skipping...
WARNING: net-tools-1.60+git20180626.aebd88e-r0 do_package: net-tools: alternative target (/bin/domainname or /bin/domainname.net-tools) does not exist, skipping...
WARNING: net-tools-1.60+git20180626.aebd88e-r0 do_package: net-tools: alternative target (/bin/hostname or /bin/hostname.net-tools) does not exist, skipping...
WARNING: net-tools-1.60+git20180626.aebd88e-r0 do_package: net-tools: alternative target (/bin/nisdomainname or /bin/nisdomainname.net-tools) does not exist, skipping...
WARNING: net-tools-1.60+git20180626.aebd88e-r0 do_package: net-tools: alternative target (/bin/ypdomainname or /bin/ypdomainname.net-tools) does not exist, skipping...
WARNING: net-tools-1.60+git20180626.aebd88e-r0 do_package: net-tools: NOT adding alternative provide /usr/share/man/man1/hostname.1: /usr/share/man/man1/hostname.1.net-tools does not exist
WARNING: net-tools-1.60+git20180626.aebd88e-r0 do_package: net-tools: NOT adding alternative provide /usr/share/man/man1/dnsdomainname.1: /usr/share/man/man1/dnsdomainname.1.net-tools does not exis$
WARNING: net-tools-1.60+git20180626.aebd88e-r0 do_package: net-tools: NOT adding alternative provide /sbin/arp: /sbin/arp.net-tools does not exist
WARNING: net-tools-1.60+git20180626.aebd88e-r0 do_package: net-tools: NOT adding alternative provide /sbin/ifconfig: /sbin/ifconfig.net-tools does not exist
WARNING: net-tools-1.60+git20180626.aebd88e-r0 do_package: net-tools: NOT adding alternative provide /sbin/plipconfig: /sbin/plipconfig.net-tools does not exist
WARNING: net-tools-1.60+git20180626.aebd88e-r0 do_package: net-tools: NOT adding alternative provide /sbin/rarp: /sbin/rarp.net-tools does not exist
WARNING: net-tools-1.60+git20180626.aebd88e-r0 do_package: net-tools: NOT adding alternative provide /sbin/route: /sbin/route.net-tools does not exist
WARNING: net-tools-1.60+git20180626.aebd88e-r0 do_package: net-tools: NOT adding alternative provide /sbin/slattach: /sbin/slattach.net-tools does not exist
WARNING: net-tools-1.60+git20180626.aebd88e-r0 do_package: net-tools: NOT adding alternative provide /bin/dnsdomainname: /bin/dnsdomainname.net-tools does not exist
WARNING: net-tools-1.60+git20180626.aebd88e-r0 do_package: net-tools: NOT adding alternative provide /bin/domainname: /bin/domainname.net-tools does not exist
WARNING: net-tools-1.60+git20180626.aebd88e-r0 do_package: net-tools: NOT adding alternative provide /bin/hostname: /bin/hostname.net-tools does not exist
WARNING: net-tools-1.60+git20180626.aebd88e-r0 do_package: net-tools: NOT adding alternative provide /bin/nisdomainname: /bin/nisdomainname.net-tools does not exist
WARNING: net-tools-1.60+git20180626.aebd88e-r0 do_package: net-tools: NOT adding alternative provide /bin/ypdomainname: /bin/ypdomainname.net-tools does not exist
WARNING: net-tools-1.60+git20180626.aebd88e-r0 do_package: net-tools: alt_link == alt_target: /usr/share/man/man1/hostname.1 == /usr/share/man/man1/hostname.1
WARNING: net-tools-1.60+git20180626.aebd88e-r0 do_package: net-tools: alt_link == alt_target: /usr/share/man/man1/dnsdomainname.1 == /usr/share/man/man1/dnsdomainname.1
WARNING: net-tools-1.60+git20180626.aebd88e-r0 do_package: net-tools: alt_link == alt_target: /sbin/arp == /sbin/arp
WARNING: net-tools-1.60+git20180626.aebd88e-r0 do_package: net-tools: alt_link == alt_target: /sbin/ifconfig == /sbin/ifconfig
WARNING: net-tools-1.60+git20180626.aebd88e-r0 do_package: net-tools: alt_link == alt_target: /sbin/plipconfig == /sbin/plipconfig
WARNING: net-tools-1.60+git20180626.aebd88e-r0 do_package: net-tools: alt_link == alt_target: /sbin/rarp == /sbin/rarp
WARNING: net-tools-1.60+git20180626.aebd88e-r0 do_package: net-tools: alt_link == alt_target: /sbin/route == /sbin/route
WARNING: net-tools-1.60+git20180626.aebd88e-r0 do_package: net-tools: alt_link == alt_target: /sbin/slattach == /sbin/slattach
WARNING: net-tools-1.60+git20180626.aebd88e-r0 do_package: net-tools: alt_link == alt_target: /bin/dnsdomainname == /bin/dnsdomainname
WARNING: net-tools-1.60+git20180626.aebd88e-r0 do_package: net-tools: alt_link == alt_target: /bin/domainname == /bin/domainname
WARNING: net-tools-1.60+git20180626.aebd88e-r0 do_package: net-tools: alt_link == alt_target: /bin/hostname == /bin/hostname
WARNING: net-tools-1.60+git20180626.aebd88e-r0 do_package: net-tools: alt_link == alt_target: /bin/nisdomainname == /bin/nisdomainname
WARNING: net-tools-1.60+git20180626.aebd88e-r0 do_package: net-tools: alt_link == alt_target: /bin/ypdomainname == /bin/ypdomainname
```

**Comment by AkitoTakada** [comment]
Removed do_patch as pointed out.
Also addressed warning.

**Comment by masami256** [comment]
thanks!

**Pull request merged**